### DISCUSSION TITLE
Alert Component not compiling correctly

### DISCUSSION THREAD

**Discussion by destradaarias** [twilio-labs/paste#628] [Q&A] [answered]
Whenever I install the `@twilio-paste/alert` component and render `<Alert>hello world</Alert>`, I get a number of "Module Not Found" errors, such as 
`Module not found: Error: Can't resolve 'react-uid' in '/Users/destradaarias/Projects/notification-preferences-console/node_modules/@twilio-paste/alert/dist'
`

Any ideas?

**Comment by richbachman** [comment, marked as answer]
Hi @destradaarias. It looks like you might just be missing the `react-uid` external dependancy. If you install that, does that solve your issue?

You can see a list of all the dependencies here if you want to verify you have them all installed. https://paste.twilio.design/getting-started/engineering

**Reply by destradaarias** [reply]
Ah yes, most of the dependencies listed on that webpage were missing. tysm!
